cli/kv.go

```diff
@@ -187,7 +189,9 @@ var delCmd = &cobra.Command{
 	Use:   "del [options] <key> [<key2>...]",
 	Short: "deletes the value for a key",
 	Long: `
-Deletes the value for one or more keys.
+Deletes the value for one or more keys. Only two keys are expected when
+--range flag is used. If the --range flag is used then all the keys in
+range of <key1> and <key2> is deleted including <key1> and excluding <key2>.
```

Comment:
`s/range/the range/g`

I would have used the standard notation of open and closed ranges. Something like: `the range [<key1>, <key2>)`, but perhaps that is too terse.